### PR TITLE
New How-to hydraulic modelling

### PR DESCRIPTION
**Pull request by urskaufmann**
Hydraulic geometry, overflows, overflow characteristic: how to use for easy handling of the date in Mike Urban/Mike+

### CODE REVIEW

File: en/user-guide/How-To/index.rst [hunk 1]
```diff
@@ -62,3 +62,109 @@ If you can not find a .qgd-file, then you have probably not moved or rotated a l
...
+
+2. Select the wastewater nodes tab in the Feature Attributes window
+
+3. Select a hydr_geometry in the fk_hydr_geometry field or use the + button to create a new hydr_geometry.
+
+.. figure:: images/hydr_geometry1.jpg
+
+4. Enter a description in the hydr_geometry - Feature Attributes window. This name is also the name of the table in MikeUrban.
+
+5. Before you can create a hydr: geom_relation, the new record hydro_geometry must be saved: -> click OK
```
Comment (sjib) [inline en/user-guide/How-To/index.rst:85]: 5. Before you can create a hydr: geom_relation, the new record hydr_geometry must be saved: -> click OK

File: en/user-guide/How-To/index.rst [hunk 2]
```diff
@@ -62,3 +62,109 @@ If you can not find a .qgd-file, then you have probably not moved or rotated a l
...
+
+.. figure:: images/hydr_geometry2.jpg
+
+7. With the **Add child object** button, you can now generate the records with which the hydr_geometry is defined analogously to the basin geometry of MikeUrban (H, As surface, Ac cross-sectional area).
+
+.. figure:: images/hydr_geometry3.jpg
+
+Note:
+
+* The water depth is the value above the bottom level or the outlet. A hydr_geometry can thus be used for several wastewater structure if they are similar built.
```
Comment (sjib) [inline en/user-guide/How-To/index.rst:97]: * The water depth is the value above the bottom level or the outlet. A hydr_geometry can thus be used for several wastewater structure if they are built similarly .

File: en/user-guide/How-To/index.rst [hunk 3]
```diff
@@ -62,3 +62,109 @@ If you can not find a .qgd-file, then you have probably not moved or rotated a l
...
+
+In the next picture, the 2nd node is marked in yellow on the map and in the Feature Attributes window.
+
+.. figure:: images/overflow3.jpg
+
+We define now a prank weir:
+
+6. Choose prank weir in the overflow tabs
+
+7. Create a weir with the * Add child object * button
```
Comment (sjib) [inline en/user-guide/How-To/index.rst:132]: 7. Create a weir with the **Add child object** button

File: en/user-guide/How-To/index.rst [hunk 4]
```diff
@@ -62,3 +62,109 @@ If you can not find a .qgd-file, then you have probably not moved or rotated a l
...
+3. Click on the wastewater node
+
+4. Confirm that the connection is created for the **from reach point**.
+
+.. figure:: images/connect_2_node.jpg
+
+Overflow characteristic
+-----------------------
+
+In the case of a leaping weir, a pumpt or under special conditions, an overflow characteristic can be defined for the overflow:
```
Comment (sjib) [inline en/user-guide/How-To/index.rst:158]: In the case of a leaping weir, a pump or under special conditions, an overflow characteristic can be defined for the overflow:

File: en/user-guide/How-To/index.rst [hunk 5]
```diff
@@ -62,3 +62,109 @@ If you can not find a .qgd-file, then you have probably not moved or rotated a l
...
+
+You can select an existing characteristic in the field fk_overflow_char or you creat a new one characteristic with the green + button.
+
+The Feature Attributes window for the overflow characteristic appears:
+
+.. figure:: images/overflow_char1.jpg
+
+1. Here, too, the identifer is later adopted in MikeUrban as a table name for a QH relationship, which is used, for example, in a local controller.
```
Comment (sjib) [inline en/user-guide/How-To/index.rst:166]: 1. Here too, the identifer is later adopted in MikeUrban as a table name for a QH relationship, which is used, for example, in a local controller.

File: en/user-guide/How-To/index.rst [hunk 6]
```diff
@@ -62,3 +62,109 @@ If you can not find a .qgd-file, then you have probably not moved or rotated a l
...
+
+2. Because the new record overflow characteristic has not yet been saved, it must first be saved with OK before the records of the HQ relation can be created.
+
+3. Go with the button **switch to form view** in the field fk_ overflow characteristic back to the just created overflow characteristic and define the necessary HQ or QQ values with the **add child object** button.
```
Comment (sjib) [inline en/user-guide/How-To/index.rst:170]: 3. Go back in the field fk_ overflow characteristic of the just created overflow characteristic with the button **switch to form view**  and define the necessary HQ or QQ values with the **add child object** button.

**Review by sjib — APPROVED** [review]

**Comment by sjib** [comment]
@m-kuhn Von mir aus ok, kann ich das mergen?

**Comment by m-kuhn** [comment]
Ja, gerne

**Pull request merged**